cpu/stm32f4/vectors.c

```diff
@@ -103,11 +105,20 @@ WEAK_DEFAULT void isr_dma2_stream7(void);
 WEAK_DEFAULT void isr_usart6(void);
 WEAK_DEFAULT void isr_i2c3_ev(void);
 WEAK_DEFAULT void isr_i2c3_er(void);
+#if defined(CPU_MODEL_STM32F413ZH)
```

Comment:
Finally, I think the specific defined is not required. I tested it for another CPU (F0) and it still builds if the function is not used.
I don't know what's the best practice here. Maybe @haukepetersen can help ?